source/migrating_from_legacy_products/index.html.md.erb

```diff
@@ -3,6 +3,121 @@ title: Migrate from our old frameworks
 weight: 210
 ---
 
-# Migrate from our old frameworks
+# Migrate to GOV.UK Frontend from our old frameworks
 
-Read about how to [update from GOV.UK Elements and Frontend Toolkit](https://test_moving_docs--govuk-design-system-preview.netlify.com/get-started/updating-your-code/).
+You should migrate to GOV.UK Frontend if you're using one of the following old ('legacy') frameworks:
+
+- GOV.UK Frontend Toolkit
+- GOV.UK Template
+- GOV.UK Elements
+
+We're no longer updating our old frameworks, and they do not meet the [Web Content Accessibility Guidelines (WCAG 2.1) accessibility standard](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps).
+
+After you include GOV.UK Frontend in your project, your service should look and behave the same as before. This means you can migrate one component at a time in your live service.
+
+Your service may be slightly slower while you migrate, because your users will need to download more CSS and JavaScript.
+
+## Audit your service's components
```

Comment:
This section feels more relevant for the 'migrate on at a time' should we consider moving it?

Reply:
@36degrees - what do you think?

Reply:
Unsure. I think we want to encourage users to at least consider thinking about their service in terms of components regardless of how they update. I think this and the section about using macros is relevant to both, although it is less important for the smaller services.